odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1304,6 +1304,8 @@ def _postprocess_tag_field(self, node, name_manager, node_info):
                 if isinstance(domain, str):
                     vnames = get_expression_field_names(domain)
                     name_manager.must_have_fields(node, vnames, node_info, ('domain', domain))
+            if field.type == 'properties':
+                name_manager.must_have_fields(node, [field.definition_record], node_info, ('properties', field.definition_record))
```

Comment:
Not sure what I should put as use instead of `('properties', field.definition_record)`